README.md

```diff
@@ -1,31 +1,29 @@
-# Play! 2.3.X plugin for Google Closure Templates
-
-This plugin is designed for using Google Closure Templates with Play! 2.3.X
-
-## Prerequisites
-
-The latest Google Closure Templates jar from google: http://code.google.com/p/closure-templates/
-
-Install the latest sbt-closure-templates plugin: https://github.com/gawkermedia/sbt-closure-templates
+# Play! 2.3.x plugin for Google Closure Templates
 
+This plugin is designed for using Google Closure Templates with Play! 2.3.x
 
 ## Installation
 
-1. in ```project/Build.scala``` add ```"com.kinja.play" %% "play2-closure" % "0.41-2.3.4-SNAPSHOT"``` to your ```project/Build.scala``` file's ```app dependencies``` section.
+1. Clone the repository
```

Comment:
We could publish this to Maven Central as with our other open-source projects. I haven't done so due to concerns about the quality of the interface and uncertainty about the best way to version a Play plugin;,but if people are actually using this (as a recent issue indicates they are), that is the correct thing to do.